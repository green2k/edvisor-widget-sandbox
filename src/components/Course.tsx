import * as React from 'react'

export const Course = ({name, description, price, ageMin, ageMax}) => (
	<div>
		<h1>{name}</h1>
		<div>Description: {description}</div>
		<div>Price: {price}</div>
		<div>Age: {ageMin} - {ageMax}</div>
	</div>
);

export default Course
