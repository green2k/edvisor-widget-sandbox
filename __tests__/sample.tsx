import * as React from 'react'
import * as renderer from 'react-test-renderer';

import Course from '../src/components/Course'

describe('<Course />', () => {
  it('renders', () => {
  	const tree = renderer.create(<Course name="Course 1" />).toJSON()
  	expect(tree).toMatchSnapshot()
  })
});
