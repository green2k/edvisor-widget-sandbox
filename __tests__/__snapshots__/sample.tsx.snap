// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Course /> renders 1`] = `
<div>
  <h1>
    Course 1
  </h1>
  <div>
    Language: 
  </div>
  <div>
    Description: 
  </div>
  <div>
    Price: 
  </div>
  <div>
    Age: 
     - 
  </div>
</div>
`;
